diff --git a/Users/tdurieux/git/dinghy-experiment/data/evaluation/dockerfiles/google/google-ctf/2022/sandbox-treebox/challenge/Dockerfile b/Users/tdurieux/git/dinghy-experiment/data/evaluation/dockerfile_repair_results/google/google-ctf/2022/sandbox-treebox/challenge/Dockerfile/repaired.Dockerfile
index c1e2506..8452927 100644
--- a/Users/tdurieux/git/dinghy-experiment/data/evaluation/dockerfiles/google/google-ctf/2022/sandbox-treebox/challenge/Dockerfile
+++ b/Users/tdurieux/git/dinghy-experiment/data/evaluation/dockerfile_repair_results/google/google-ctf/2022/sandbox-treebox/challenge/Dockerfile/repaired.Dockerfile
@@ -22,7 +22,7 @@ RUN chmod a+x /home/user/treebox.py
 
 # If you see an error on this line it might be due to an old version of Docker.
 # Ensure you have the latest Docker version.
-RUN apt-get update && apt-get install -y python3
+RUN apt-get update && apt-get install --no-install-recommends -y python3 && rm -rf /var/lib/apt/lists/*;
 
 FROM gcr.io/kctf-docker/challenge@sha256:d884e54146b71baf91603d5b73e563eaffc5a42d494b1e32341a5f76363060fb